.github/dependabot.yml

```diff
@@ -1,38 +1,89 @@
 version: 2
 updates:
+  # ==============
+  # KOTLIN BACKEND
+  # ==============
+
+  # Kjører sjekk hver måned for generelle oppdateringer
   - package-ecosystem: "gradle"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "monthly" 
+      day: "sunday"
+    open-pull-requests-limit: 10
+
+  # Daglig sjekk (limit=0) for å sikre sikkerhetsoppdateringer
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily" 
+    open-pull-requests-limit: 0
```

Comment:
Kunne det like gjerne stått "5" her da?

Reply:
_This option has no impact on security updates, which have a separate, internal limit of ten open pull requests._

Reply:
Må ikke settes til 5. Da får vi vanlige bumps igjen. Dette er en hack for å tvinge den til å utelukkende ta sikkerhetsoppdateringer.